bin/node/src/service.rs

```diff
@@ -390,11 +399,12 @@ pub fn new_authority(
     if aleph_config.external_addresses().is_empty() {
         panic!("Cannot run a validator node without external addresses, stopping.");
     }
-    let blockchain_backend = BlockchainBackendImpl { backend };
+    let chain_status = SubstrateChainStatus::new(backend)
```

Comment:
nit: I don't like the idea that `SubstrateChainStatus` implements also `JustificationTranslator`. Relation looks more like: the only implementation of `JustificationTranslator` uses impl of `ChainStatus`. The `JustificationTranslator` trait also doesn't look very useful at the moment - only single implementation (shared with some other impl by SubstrateChainStatus), no mocks.

Reply:
Yeah, I'm not sure what to do about this. I originally wanted to implement this for everything that implements `ChainStatus`, but that obviously doesn't work for typical reasons. :/ If you have some specific better idea I would be grateful, but preferably not in this PR (unless it would make it smaller?). This also related to the last comment ("Maybe don't merge them?").